cmake/find/FindLive555.cmake

```diff
@@ -0,0 +1,223 @@
+# FindLive555
+#
+# Find the Live555 Streaming Media Libraries.
+#
+#=============================================================================
+# Copyright 2006-2009 Kitware, Inc.
+# Copyright 2009-2011 Mathieu Malaterre <mathieu.malaterre@gmail.com>
+# Copyright 2015 Alexander Lamaison <alexander.lamaison@gmail.com>
+# Copyright 2017 Ernesto Varoli <ernesto.varoli@gmail.com>
+#
+# Distributed under the OSI-approved BSD License (the "License");
+# see accompanying file Copyright.txt for details.
+#
+# This software is distributed WITHOUT ANY WARRANTY; without even the
+# implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the License for more information.
+#=============================================================================
+# (To distribute this file outside of CMake, substitute the full
+#  License text for the above reference.)
+
+if(HUNTER_STATUS_DEBUG)
+  message("[hunter] Custom FindLive555 module")
+endif()
+
+# Support preference of static libs by adjusting CMAKE_FIND_LIBRARY_SUFFIXES
+if(LIVE555_USE_STATIC_LIBS)
+  set(_live555_ORIG_CMAKE_FIND_LIBRARY_SUFFIXES ${CMAKE_FIND_LIBRARY_SUFFIXES})
+  if(WIN32)
+    set(CMAKE_FIND_LIBRARY_SUFFIXES .lib .a ${CMAKE_FIND_LIBRARY_SUFFIXES})
+  else()
+    set(CMAKE_FIND_LIBRARY_SUFFIXES .a )
+  endif()
+endif()
+
+find_path(LIVE555_GROUPSOCK_INCLUDE_DIR
```

Comment:
Since we are expecting headers in `LIVE555_ROOT` you can set path explicitly to `${LIVE555_ROOT}/include/` and check if `${LIVE555_ROOT}/include/live555(?)/Groupsock.hh` exists.